client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -114,7 +117,13 @@ export function receiveError( store ) {
 	);
 }
 
-export default {
+const followingMine = {
```

Comment:
this looks weird, but it's because we have a case where we have nested API urls.

`/read/following/mine` is the set of things you follow ( and this file )
`/read/following/mine/new` makes a new follow
`/read/following/mine/delete` remove a follow

hence the somewhat weird file structure.